components/feature/qr/src/main/java/mozilla/components/feature/qr/QrFragment.kt

```diff
@@ -127,7 +128,11 @@ class QrFragment : Fragment() {
         override fun onOpened(cameraDevice: CameraDevice) {
             cameraOpenCloseLock.release()
             this@QrFragment.cameraDevice = cameraDevice
-            createCameraPreviewSession()
+            try {
+                createCameraPreviewSession()
+            } catch (e: CameraAccessException) {
+                Log.log(Log.Priority.ERROR, "mozac-qr", e, message = "Failed to create camera preview session")
```

Comment:
nit: You have a bunch of those Log calls. Using `Logger` may be more comfortable. :)

Reply:
Ah, yes, will update. Desperately trying to write a test for this too :)